README.md

```diff
@@ -1,2 +1,72 @@
-# Capeston-Project
-In this capstone project I am going to build is based on an online website for a conference
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone js.
+
+> In this branch (capstone__js) 😊 I made the two pages (The home page. The about page). When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.The section "Featured speakers" created dynamically in JavaScript and I made the project responsive.👌
+
+## Built With
+
+- HTML
+- Use semantic HTML tags.
+- Apply best practices in HTML code.
+- CSS
+- Use CSS selectors correctly.
+- Use CSS box model.
+- Use Flexbox to place elements in the page.
+- (RWD) responsive web design using media query
+- Animation and Transition
+- Use GitHub Pages to deploy web pages.
+- Check the accessibility for this portfolio
+- Apply JavaScript best practices and language style guides in code.
+- Use JavaScript to manipulate DOM elements.
+- Use JavaScript events.
+- Use objects to store and access data.
+- Communicate technical concepts to other technical people.
+
+## Live Demo
+[Capstone-Project](https://mrkamin.github.io/Capeston-Project/)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+## Loom video to explain the capstone project
+
+> [click here 😊 to watch the video](https://www.loom.com/share/a3e5555abacd4e5d94087814355474c5)
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/mrkamin)
+- Twitter: [@twitterhandle](https://twitter.com/Mohamma63974237)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammad-rafi-amin-63b4319b/)
```

Comment:
- [OPTIONAL] It would be nice and make your README looks even more professional if you change the example in the 'Getting started' section and add your name in the 'Authors' section and after the Github, Twitter and Linkedin.